.github/workflows/knative-style.yaml

```diff
@@ -60,3 +60,13 @@ jobs:
               exit 1
           fi
           echo "${{ github.repository }} is formatted correctly."
+      
+      - name: Language
+        shell: bash
+        run: |
+          changed=$(git diff --name-only HEAD HEAD~1 | grep -v '/vendor\|/third-party\|.github\|go.sum\|go.mod\|.md')
+          if [ ! -z "$changed" ]; then
```

Comment:
You can also look for introductions of new files with unwanted terms. E.g., a new file called 'whitelist.txt'